pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/DispatchRate.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.common.policies.data;
+
+public class DispatchRate {
+
+    public long topicDispatchRate = -1;
+    public DispatchRateType dispatchRateTye = DispatchRateType.messageRate;
+
+    public enum DispatchRateType {
+        messageRate, byteRate;
+    }
+
+    public DispatchRate() {
+        this.topicDispatchRate = -1;
+        this.dispatchRateTye = DispatchRateType.messageRate;
+    }
+
+    public DispatchRate(long topicMessageRate, DispatchRateType dispatchRateTye) {
+        super();
+        this.topicDispatchRate = topicMessageRate;
+        this.dispatchRateTye = dispatchRateTye;
+    }
+
+    @Override
+    public int hashCode() {
```

Comment:
Unless `hashCode()` is used in a very critical place, just use `Objects.hashCode(var1, var2)`